spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingIgnoringErrors.scala

```diff
@@ -0,0 +1,55 @@
+package za.co.absa.spark.commons.errorhandling.implementations
+
+import org.apache.spark.sql.functions.lit
+import org.apache.spark.sql.types.{ DataType, NullType}
+import org.apache.spark.sql.{Column, DataFrame}
+import za.co.absa.spark.commons.errorhandling.{ErrorHandling, ErrorMessageSubmit}
+
+object ErrorHandlingIgnoringErrors extends ErrorHandling {
+
+  /**
+   * First of the few methods that needs to be coded in the trait implementation
+   * The purpose of this method is to convert the error specification into a [[org.apache.spark.sql.Column spark.Column]] expression
+   *
+   * @param errorMessageSubmit - the error specification
+   * @return - the error specification transformed into a column expression
+   * @group Error Handling
+   * @since 0.6.0
+   */
+  override protected def transformErrorSubmitToColumn(errorMessageSubmit: ErrorMessageSubmit): Column = {
+    lit(null)
```

Comment:
This works, but let's use `null_col` from `za.co.absa.spark.commons.sql.functions` as suggested. 😉 
It avoids using the `null` value in the code, which some linters might mark as an issue.

Reply:
I'm on it.